<?xml version="1.0" ?>

<launch>        
    <param name="gazebo"                   value="false"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_humanoid"/>
    
    <!-- <param name="offset_table"             value="$(find op3_manager)/config/offset.yaml"/> -->
    <param name="robot_file_path"          value="$(find op3_sensors)/config/OP3_SENSOR.robot"/>
    <param name="init_file_path"           value="$(find op3_sensors)/config/dxl_init.yaml"/>
        	 
    
    <!-- THORMANG3 Manager -->
    <node name="op3_cm_740" pkg="op3_sensors" type="op3_sensors" output="screen"/>
    
</launch>
